BlueprintUI/Sources/Layout/Hidden.swift

```diff
@@ -0,0 +1,45 @@
+import CoreGraphics
+
+/// `Hidden` conditionally hides its wrapped element.
+///
+/// - Note: When an element is hidden, any elements within the wrapped element will be hidden.
+public struct Hidden: Element {
```

Comment:
Do we want to go with `IsHidden`, maybe?

Reply:
We already have `UserInteractionEnabled` (without `is`).

I'm fine with this (or updating both + any others that are similar if they exist)—I'd mainly just want to be consistent.

Reply:
I prefer `Hidden`, I think it's a better name since it's an element and not a boolean property.